README.md

```diff
@@ -37,7 +37,9 @@ Call the `predict_for_protein` function to get predictions for your docked prote
 
 ```python
 from pathlib import Path
+from hydrascreen.predictor import HydraScreen
 
+predictor = HydraScreen("eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InRlc3RAZW1haWwuY29tIiwib3JnIjoiTXkgT3JnIiwiZXhwIjoxNjk1NDYyMzU3fQ.Wwm6TBud41FnLc_2XZMa-ws_j7BjKY3fCwBzRKzaVvc") # replace with token received from email
```

Comment:
This needs to be aligned with the backend email sending. Either it has to send a raw token for Python users or this one needs to be base64 encoded.

Reply:
Also, maybe worth writing `HydraScreen("<TOKEN>")` instead of an example value?

Reply:
nice catch!

Reply:
prefer to leave an example of token so they know what to expect